db/schema.rb

```diff
@@ -11,15 +11,30 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160127035545) do
+ActiveRecord::Schema.define(version: 20160127193752) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "games", force: :cascade do |t|
```

Comment:
So it looks like there are conflicts regarding `Games` which should never have appeared on this branch (but I can guess that it's now on the branch due to the migration errors I had from before). To remove this, should I just edit my schema by deleting the `Games` table? Or should the schema NEVER be edited.